api.py

```diff
@@ -18,4 +18,4 @@ class ScoringItem(BaseModel):
 async def scoring_endpoint(item:ScoringItem): 
     df = pd.DataFrame([item.dict().values()], columns=item.dict().keys())
     yhat = model.predict(df)
-    return {"prediction":int(yhat)}
\ No newline at end of file
+    return {"prediction":str(yhat)}
```

Comment:
add new line

Reply:
can you give an example of what are you meaning?

Reply:
<img width="41" alt="image" src="https://github.com/dquerales/FastAPI/assets/51675103/8fa458ba-de1d-4b5b-9209-fb888bad15b2">

this red circle with a hyphen inside of it indicates that the file doesn't end with an EOL or end of line. Different computer systems use different line ending styles. Some use a simple newline character (\n) such as Linux, while others use this one (\r\n) like Windows. Adding that EOL makes sure your code plays nice on all systems with all kind of tools. It's also a way to ensure consistency and promote good practices among all files

Reply:
you just need to add an empty new line after the last line of code

Reply:
it would be a good idea to implement a linter into a github action to validate that all files are consistent across the project. This way, we can define a unique style to improve consistency and readability

Reply:
Sure. I am about to add a new github action for this